tests/test_training_validation.py

```
@@ -0,0 +1,676 @@
+
+    CUSTOM_FIELDS = {
+        "others_lr",
+        "others_weight_decay",
+        "focal_loss_alpha",
+        "focal_loss_gamma",
+        "focal_loss_prob_margin",
+        "label_smoothing",
+        "loss_reduction",
+        "negatives",
+        "masking",
+    }
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)